### DISCUSSION TITLE
Cancel show rule for table's first row

### DISCUSSION THREAD

**Discussion by nmayorov** [typst/typst#3831] [Q&A] [answered]
I've written `#show table.cell.where(y: 0): strong` and happily used it for the most of my document.

But now I wan't to render a special table with a normal text in the first row. Writing `#show table.cell.where(y: 0): it => it` doesn't change anything.

Is it possible to achieve that in some reasonable way? 

Typst version: 0.11.

**Comment by laurmaedje** [comment, marked as answer]
You can write `#show table.cell.where(y: 0): set text(weight: "bold")` instead of via `strong` and then go back with `#show table.cell.where(y: 0): set text(weight: "regular")`.

**Comment by nmayorov** [comment]
Thank you, great advice.

Looking forward for further Typst development and improvements.
